Server/src/routes/fileParserRouter.ts

```diff
@@ -26,4 +27,4 @@ router.post('/api/v1/dataExtract', [JWTAuthenticator.verifyJWT, upload.single('f
   }
 })
 
-export { router as dataExtractionRouter };
\ No newline at end of file
+export { router as fileParserRoute };
```

Comment:
Why was the name of the route changed? Further, even if the change is needed, why weren't the controller, main service, and the name of subdirectory containing all the dataExtraction services changed too?

Reply:
All dataExtraction code were merged with fileParser instead to remove duplicate code.

Reply:
That doesn't answer why the controller and service were not also renamed. Solely renaming the route has made the naming inconsistent and confusing. 
Additionally, I don't see any sort of duplicate code that was removed and so I am confused by this point. Can you elaborate?

Reply:
[Check here](https://github.com/Data-Boom/SOEN490/pull/486/commits/e116e1df5d26807b57316dd62ae87a436ac67ab2)
We had fileParser and dataExtraction that basically did the same thing with different routes and controllers. Now everything related to parsing will be done by fileParserRoute

Reply:
I see, that makes sense. I also see, since the comment was originally made, the service and other parts were renamed as requested. Awesome work!